dev-shell

```diff
@@ -9,7 +9,7 @@ cd $here
 
 if ! docker image inspect $container 2>/dev/null 1>/dev/null ; then
 	echo "Container not found, doing one time docker build"
-	docker build -t $container -f ./aktualizr/docker/Dockerfile.debian.testing .
+	docker build -t $container -f ./docker/Dockerfile.ubuntu.bionic .
```

Comment:
let's see what the aktualizr team will do about it if anything, I would stay with whatever they will come up with because if we just switch to something higher we might end-up with some incompatibilities as you stated. I consider it as a temporal solution.